### DISCUSSION TITLE
UV Tools make Solide Resin Block at print

### DISCUSSION THREAD

**Discussion by CaptainGelb** [sn4k3/UVtools#675] [General]
Hi @ all,
i am currently working with Chitubox. If i slice my file and use UV tools to manipulate it, the printer just fills all layers and prints me a solid resin block. 
I simply didn't fix errors in UV tools but i would rly like to. 

I am currently using: 
- chitubox 1.8.1 
- Latest Version of UV tools
- Elegoo Mars (standard version)

**Comment by sn4k3** [comment]
Using cbddlp file?

**Reply by CaptainGelb** [reply]
Yes thats right.

**Reply by sn4k3** [reply]
Can zip and attach file here?

**Reply by CaptainGelb** [reply]
[files.zip](https://github.com/sn4k3/UVtools/files/11014458/files.zip)

Sure here are the files bevore and after the edit.

**Reply by sn4k3** [reply]
Does `Source File` prints correctly?

**Reply by sn4k3** [reply]
Can you try to dry print this file processed by UVtools?

[Source File_UVtools.zip](https://github.com/sn4k3/UVtools/files/11020160/Source.File_UVtools.zip)

**Reply by CaptainGelb** [reply]
Yes the Source File prints fine.

Yes I ll give it a try.

**Reply by CaptainGelb** [reply]
your file reads correct. The Printer doesn't try to pront a solid block.

**Reply by sn4k3** [reply]
Ok, found the issue then. WIll fix on next patch. Thanks
